ArcGISRuntimeSDKQt_CppSamples/Scenes/ChooseCameraController/ChooseCameraController.qrc

```diff
@@ -0,0 +1,11 @@
+<RCC>
+    <qresource prefix="/Samples/Scenes/ChooseCameraController">
+        <file alias="sample.json">ChooseCameraController.json</file>        
+        <file>ChooseCameraController.qml</file>
+        <file>ChooseCameraController.h</file>
+        <file>ChooseCameraController.cpp</file>
+		<file>main.qml</file>
```

Comment:
alignment. I think if you update your samples JSON templates this has been addressed